scripts/inspect_replay_and_td.py

```
@@ -0,0 +1,302 @@
+
+    if state_size is None:
+        state_size = 82
+
+    agent = load_agent(model_path, state_size)
+
+    if replay_path:
+        replay = load_replay(replay_path)
+    elif dataset_features is not None and close_prices is not None:
+        replay = build_replay_from_dataset(dataset_features, close_prices)
+    else:
+        raise ValueError("Need either dataset_path or replay_path to evaluate checkpoint")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)